lektor/imagetools.py

```diff
@@ -359,14 +358,6 @@ def find_imagemagick(im=None):
     # convert is system utility for fs manipulation.
     imagemagick_exe = 'convert' if os.name != 'nt' else 'magick'
 
-    # If we have a shipped imagemagick, then we used this one.
```

Comment:
no, it's because all that logic is already handled in [`locate_executable`](https://github.com/xlotlu/lektor/blob/6268e90e551004fe2e466bd66d9f674f3eb3f5aa/lektor/utils.py#L383), albeit slightly different, with `PATHEXT` and the `for path in choices: for ext in extensions:` part.